projects/layout/src/modules/box/box-content-section.component.ts

```diff
@@ -0,0 +1,8 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'sky-box-content-section',
```

Comment:
☝️ I noticed tiles have a `<sky-tile-content>` that wraps multiple `<sky-tile-content-section>` elements - each one with its own padding and border. We don't have a concept of that type of nesting here. Should this just be called `<sky-box-content>`, or is the word "section" helpful?

Reply:
Good question. We also use `sky-modal-content` for the modal component. I'm fine with that, if Adam doesn't have an opinion either way.

Reply:
Let's skip the nesting of sections inside of content for Box. Talked to Todd about this and he's in agreement that we can let consumers DIY the spacing that modern "sections" would use anyways.

`sky-box-content`

Reply:
Updated to use `sky-box-content`.